app/models/user.rb

```diff
@@ -1,5 +1,13 @@
 class User < ApplicationRecord
-  has_many :posts, dependent: :destroy
-  has_many :likes, dependent: :destroy
-  has_many :comments, dependent: :destroy
+  has_many :comments
+  has_many :posts
+  has_many :likes
```

Comment:
- [ ] Let us establish these associations correctly. With this current set up when we run `user.posts` for example Rails will search for posts using the `user_id` column, but what we have is `author_id` (from the recommendation above). Kindly fix these associations and highlight which column rails should look at.

  ```
  has_many :posts, foreign_key: 'author_id'
  has_many :comments, foreign_key: 'author_id'
  has_many :likes, foreign_key: 'author_id'
  ```